src/components/panel/panel.js

```diff
@@ -945,13 +960,24 @@ MdPanelRef.prototype.show = function() {
   // TODO(ErinCoughlan) - Cancel any in-progress actions.
 
   var self = this;
-  this._showPromise = this._$q(function(resolve, reject) {
+
+  var animatePromise = function() {
     self.removeClass(MD_PANEL_HIDDEN);
-    self._animateOpen().then(function() {
-      self._focusOnOpen();
-      resolve(self);
-    }, reject);
-  });
+    return self._animateOpen();
+  }
+
+  this._showPromise = this._$q.all([
+      this._backdropRef ? this._backdropRef.show() : this._$q.resolve(self),
+      animatePromise().then(function() {
+        return self._$q.resolve(self);
+      }, function() {
+        console.warn('MdPanel Animations failed. Showing panel without animating.');
```

Comment:
I think we should use $log for consistency within Material:
https://docs.angularjs.org/api/ng/service/$log